### PR TITLE
Personal Settings Setup

### PR DESCRIPTION
**Pull request by Chrissy**
defaults-5

### CODE REVIEW

File: src/sentry/static/sentry/app/views/settings/personal/personalEmails.jsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import SettingsPageHeader from '../components/settingsPageHeader';
+
+const PersonalEmails = React.createClass({
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/settings/personal/personalEmails.jsx]: Let's try to use es6 classes when possible, only use createClass if you need mixins basically.

File: src/sentry/static/sentry/app/views/settings/personal/personalNotifications.jsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+import SettingsPageHeader from '../components/settingsPageHeader';
+import personalNotificationFields from '../../../data/forms/personalNotificationSettings';
+import Form from '../components/forms/form';
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/settings/personal/personalNotifications.jsx]: You'll want `../components/forms/next/form` here

File: src/sentry/static/sentry/app/views/settings/personal/personalSettingsNavigation.jsx [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+import SettingsNavigation from '../components/settingsNavigation';
+import navigationConfiguration from './navigationConfiguration';
+
+const PersonalSettingsNavigation = React.createClass({
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/settings/personal/personalSettingsNavigation.jsx]: createClass

**Review by billyvg — COMMENTED** [review]
Looking pretty good so far

File: src/sentry/static/sentry/app/views/settings/personal/personalNotifications.jsx (hunk 2 above)
Comment (Chrissy) [reply src/sentry/static/sentry/app/views/settings/personal/personalNotifications.jsx]: ignoring this per our discussion on slack

**Comment by Chrissy** [comment]
ok @billyvg fixed changes gonna merge into `feat/personal-settings/main` which I updated to be an identical copy of https://github.com/getsentry/sentry/pull/6582 thanks for the help!

**Pull request merged**